sormas-base/setup/sormas.properties

```diff
@@ -58,19 +58,30 @@ app.url=
 # default: "/opt/sormas/custom/"
 #custom.path=/opt/sormas/custom/
 
+# Key data for the SORMAS to SORMAS feature
```

Comment:
Maybe "Configuration for the SORMAS to SORMAS feature"?
Or should this go below `#sormas2sormas.path=`?

Reply:
I wanted this to be a title for all the s2s related keys so not below  sormas2sormas.path but mark somehow that the following keys belong to sormas 2 sormas.

Do you have any suggestion how to "group" these keys?

Reply:
`Configuration for the SORMAS to SORMAS feature`?
Maybe then we won't need `# SORMAS to SORMAS truststore data` either?